data/PhyloData_ex/newton.fasta

```diff
@@ -0,0 +1,10 @@
+>0
+TAAAAGATCAATT
+>1
+CACACGTTCGAGT
+>root
+AGGACATACGTCT
+>3
+CGTAGTAGGACTC
+
+
```

Comment:
"root" represents the naive sequence (spit out by partis, for real data examples).